frontend/src/v5/ui/controls/submitButton/submitButton.component.tsx

```diff
@@ -29,7 +29,7 @@ export const SubmitButton = ({
 		fullWidth
 		variant={variant}
 		color={color}
-		style={{ marginLeft: 0, marginRight: 0 }}
+		style={{ margin: '15px 0' }}
```

Comment:
Could you please override the component using `styled(SubmitButton)` in the respective  *.style.ts files so we are more consistent?